tasks/main.yml

```diff
@@ -223,3 +223,9 @@
   become: yes
   tags:
     - xauth
+
+- name: add rhsm.certificate compile hack to .bashrc
+  lineinfile:
+    path: "{{ ansible_env.HOME }}/.bashrc"
+    line: "test -f {{ subman_checkout_dir }}/src/rhsm/*certificate*.so || (cd {{ subman_checkout_dir}}; {{ subman_python_interpreter }} setup.py build_ext --inplace 2>&1 > /dev/null)"
```

Comment:
When source code (`certificate.c`) is newer then target file `*certificate.so`, then it is also necessary to compile this file.